snappy-stubs-internal.h

```diff
@@ -60,7 +60,7 @@
 // Enable 64-bit optimized versions of some routines.
 #define ARCH_K8 1
 
-#elif defined(__ppc64__)
+#elif defined(__powerpc64__)
```

Comment:
Can you please do `#elif defined(__ppc__) || defined(__PPC__)`

I think this works as well, and matches what we have in other parts of the codebase, for example: https://github.com/abseil/abseil-cpp/blob/38b704384cd2f17590b3922b97744be0b43622c9/absl/debugging/internal/stacktrace_config.h

Reply:
sorry it took so long. Lost track of this. Suggestion works for me, though will probably defined on ppc32 however it the stubs header is also covered by HAVE_BUILTIN_CTZ (compile checked), and FindMatchLength falls back to portable version.

A test of this: https://travis-ci.org/grooverdan/snappy/builds/600582620